README.md

```diff
@@ -1,5 +1,46 @@
-# Group13_NamePending
-Group member: Son Chau
-Nikita
-yair
-Aldo
+# Quality white wine predictor
+Authors: Aldo Saltao Barros, Nikita Shimberg, Yair Guterman, Son Chou
+
+## About 
+
+**Goal**
+This project aims to determine which features are the best quality white wine indicators and generate insights into each of these factors to our model’s wine quality.
+
+**Introduction**
+According to experts, wine is differentiated according to its smell, flavour, and colour, but most people are not wine experts to say that wine is good or bad. Those variables and others not mentioned so far compose what we define as the quality of the wine. The quality of a wine is important for the consumers as well as the wine industry. For instance, industry players are using product quality certifications to promote their products. However, this is a time-consuming process and requires the assessment given by human experts, which makes this process very expensive. Nowadays, machine learning models are important tools to replace human tasks and, in this case, a good wine quality prediction can be very useful in the certification phase. For example, an automatic predictive system can be integrated into a decision support system, helping the speed and quality of the performance.
```

Comment:
Re: Those variables and others not mentioned so far compose what we define as the quality of the wine

I would rephrase this to something like "the quality of the wine is determined by many variables including, but not limited to, the ones mentioned previously".